.jenkins/pipelines/Azure/Linux/Jenkinsfile

```diff
@@ -20,6 +20,48 @@ AGENTS_LABELS = [
     "acc-rhel-8-vanilla":       env.RHEL_8_VANILLA_CUSTOM_LABEL ?: "vanilla-rhel-8"
 ]
 
+def ACCCodeCoverageTest(String label, String compiler, String build_type) {
+    stage("${label} ${compiler} SGX1FLC ${build_type} Code Coverage") {
+        node("${label}") {
+            timeout(GLOBAL_TIMEOUT_MINUTES) {
+                cleanWs()
+                checkout scm
+                def task = """
+                           sudo apt install -y lcov python3 python3-pip llvm-7
```

Comment:
Should these be moved to prereqs?

Reply:
I'll leave the decision to @BRMcLaren.  I think doing that requires updating the images.

Reply:
It would be better if we did move to prereqs, it would require an ansible change.

Reply:
I have the necessary changes for ansible already, do you want me to include them in this PR?
I would assume this also means you're going to update the image, and after that we should validate that with another bors try, correct?